data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua

```diff
@@ -200,13 +200,13 @@ obj_lumberjack_progress = {
    title=_"Let’s see the progress",
    number = 1,
    body = objective_text(_"Let’s see the progress",
-      li(_[[Click on the construction site to have a look at it, then close its window again when you have seen enough.]]) ..
-      li_arrow(_[[To close the construction site’s window, simply right-click on it.]])
+      li(_[[Click on the construction site to have a look at it.]]) ..
+      li_arrow(_[[When you have seen enough, close the construction site’s window: simply right-click on it.]])
```

Comment:
This line should still be removed. We should not suggest to the player to close the window when it needs to stay open for the workarea demonstration.

Reply:
It didn't seem inappropriate when I tested the scenario. Moreover, the objective is actually completed when player closes the construction site window.
On the other hand, nobody will probably get stuck there to seek help in Objectives window.